src/main/java/pl/szczesniak/dominik/whattowatch/movies/domain/model/AddMovieToList.java

```diff
@@ -0,0 +1,41 @@
+package pl.szczesniak.dominik.whattowatch.movies.domain.model;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+import pl.szczesniak.dominik.whattowatch.users.domain.model.UserId;
+
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+@ToString
+@EqualsAndHashCode
+public class AddMovieToList {
```

Comment:
Mam nadzieję, że wiesz, że ten builder jest zrobiony teraz na zapas, bo on w tej chwili nie jest do niczego potrzebny. Na chwilę obecną to lepiej było by mieć konstruktor do commanda, a nie buildera, bo ten builder nie ma uzasadnienia. Dopiero jak pojawią się opcjonalne pola to warto buildera zrobić. 
Ale możesz go już zostawić, bo pewnie zaraz dodamy jakieś pole